lib/main.coffee

```diff
@@ -11,16 +11,8 @@ module.exports =
       type: 'string'
       title: 'PHPMD Rulesets'
       default: 'cleancode,codesize,controversial,design,naming,unusedcode'
-      description: 'Comma separated list of rulesets to use in phpmd.'
+      description: 'Comma separated list of rulesets to use in phpmd. You can also enter the name of your ruleset file (example: `ruleset.xml`) to load that from the current file\'s directory (or any of the parent directories) '
```

Comment:
Looks like the rule for this is set to a Warning only, but can you split this into multiple lines? 120 is the max it should be allowing.